apps/music/index.html

```diff
@@ -116,7 +117,7 @@ <h1 id="title-text" data-l10n-id="music"></h1>
             <p data-l10n-id="music-paused-notification"></p>
           </section>
         </div>
-        <div id="player-seek">
+        <div id="player-seek" role="slider" aria-valuemin="0">
```

Comment:
@eeejay If the `aria-valuemin` here is to indicate the minimum value, should it be on the progress element? because we actually set the value on the progress element.